packages/web/src/scss/tools/_utilities.scss

```diff
@@ -1,9 +1,37 @@
 // Inspired by React UI and Bootstrap
+@use '@tokens' as tokens;
 @use 'sass:list';
 @use 'sass:map';
 @use 'sass:meta';
+@use 'sass:string' as sass-string;
```

Comment:
I would prefer to name our `string` not the global sass module. I think we also do this elsewhere (`spirit-string`).